wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -36,6 +36,15 @@ <h1>
         <a href="{% url 'wafer_user_profile' username=author.username %}">{{ author.userprofile.display_name }}</a>
       {% endfor %}
     </p>
+    {% if user.is_staff or perms.talks.view_all_talks %}
+      {% for author in object.authors.all %}
+        <p class='bio'>
+          {% blocktrans %}Bio{% endblocktrans %}{% if object.authors.count > 1 %} - {{author.userprofile.display_name}}{% endif %}:
+          {{author.userprofile.bio|default:"<em>Not provided</em>"}}
```

Comment:
Not provided isn't being tagged for translation. Do we care? this is not visible to the general public.

Reply:
We do care - most admin messages and other private notes are marked as translatable (not all, though, which is something that we should fix).

Reply:
ok. I couldn't find a way to translate a parameter of a filter, so I replaced it with a conditional.